lib/Cake/View/Helper/FormHelper.php

```diff
@@ -379,6 +379,13 @@ public function create($model = null, $options = array()) {
 		$this->inputDefaults($options['inputDefaults']);
 		unset($options['inputDefaults']);
 
+		if (isset($options['action'])) {
+			trigger_error('Using key `action` is deprecated, use `url` directly instead.', E_USER_DEPRECATED);
+		}
+		if (isset($options['url']) && isset($options['url']['action'])) {
```

Comment:
@dereuromark  Perhaps, is_array() check is needed here. `$options['url']` can be strings. Non-numeric string offsets didn't work as we expected in 5.3.